### PR TITLE
chm350-aichi-target view

### PR DESCRIPTION
**Pull request by liyuanmontreal**
### General description
migrate aichi-target from chm project 

### Designs
same as absch report style

### Testing instructions
choose aichi-target type report, click , show report details

image

### CODE REVIEW

File: app/app-text/views/reports/chm/aichi-target.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "fullTitle": "Full title",
+    "targetNumber": "Target Number ",
+    "strategicGoal":  "Strategic Goal",
+    "quickGuide":  "Quick Guide"
+}
```
Comment (blaisefonseca) [inline app/app-text/views/reports/chm/aichi-target.json:6]: the second word needs to be small
```suggestion
    "targetNumber": "Target number ",
    "strategicGoal":  "Strategic goal",
    "quickGuide":  "Quick guide"
}
```

File: app/views/forms/view/chm/aichi-target.vue [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                </div>
+            </div> -->
+        </div>
+    </div>
+</template>
+<script setup>
+    import { computed, onMounted } from 'vue';
+    import { lstring } from '~/services/filters/lstring.js'; 
+    import '~/components/scbd-angularjs-controls/form-control-directives/km-value-ml.js'
+    import '~/views/forms/view/directives/view-record-reference.directive.js'
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/aichi-target.vue:150]: unused import

File: app/views/forms/view/chm/aichi-target.vue [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                            <a translation-url target="{{target}}" :href="item.url">{{ item.name || item.url }}</a>                           
+                        </li>
+                    </ul>
+                </div>
+            </div> -->
+        </div>
+    </div>
+</template>
+<script setup>
+    import { computed, onMounted } from 'vue';
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/aichi-target.vue:147]: unused

File: app/views/forms/view/chm/aichi-target.vue [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    import { lstring } from '~/services/filters/lstring.js'; 
+    import '~/components/scbd-angularjs-controls/form-control-directives/km-value-ml.js'
+    import '~/views/forms/view/directives/view-record-reference.directive.js' 
+    import kmTerm from '~/components/km/KmTerm.vue';
+    import messages from '../../app-text/views/reports/chm/aichi-target.json';
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/aichi-target.vue:152]: try to use `~/` for point to a root folder

File: app/views/forms/view/record-loader.vue [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    <div>
+        <Suspense>
+            <aichi-target v-if="documentInfo?.type == 'aichiTarget'"  
+                 :document-info="documentInfo" :locale="locale"></aichi-target>
+            <!-- loading state via #fallback slot -->
+            <template #fallback>
+                Loading...
+            </template>
+        </Suspense>
+        <!-- TODO: put schema view definition element and add v-if for schema name :locale="locale" :link-target="linkTarget"-->
```
Comment (blaisefonseca) [inline app/views/forms/view/record-loader.vue:11]:
```suggestion
        <!-- TODO: put schema view definition element and add v-if for schema name -->
```

**Review by blaisefonseca — COMMENTED** [review]
Please see comments

File: app/views/forms/view/record-loader.vue (hunk 5 above)
Comment (liyuanmontreal) [reply app/views/forms/view/record-loader.vue]: fixed

File: app/views/forms/view/chm/aichi-target.vue (hunk 4 above)
Comment (liyuanmontreal) [reply app/views/forms/view/chm/aichi-target.vue]: fixed

File: app/views/forms/view/chm/aichi-target.vue (hunk 3 above)
Comment (liyuanmontreal) [reply app/views/forms/view/chm/aichi-target.vue]: fixed

File: app/views/forms/view/chm/aichi-target.vue (hunk 2 above)
Comment (liyuanmontreal) [reply app/views/forms/view/chm/aichi-target.vue]: fixed

File: app/app-text/views/reports/chm/aichi-target.json (hunk 1 above)
Comment (liyuanmontreal) [reply app/app-text/views/reports/chm/aichi-target.json]: fixed

**Pull request merged**